### PR TITLE
1436 show random fun facts on preloader

### LINKED ISSUE #1436: Preloader fun facts
Till user stares at finishing their buy into Blockchain, would be nice to maintain contact with them in reasonable way, like fun facts about NFTs is probably cheapest hack

Similar like Disocrd tricks you while they loading app in the background.

It should be extended preloader component and fetch quotes from translations in random fashion.

Let me take first round. Add them to en.json please

> Did you know officially first NFT we're Etheria back in 2015?
> Did you know that first "NFT-like" standard on Kusama are literally just byte strings? (Bruno could comment on this so we not spread something offensive but indeed funny, memorable)
> Did you know that KodaDot has started as rewrite React-based DotApps dashboard into Vuejs?
> Did you know that KodaDot is building experimental NFT landscape?
> Do you know why there is so much psychadelic and trippy NFT arts? We do not know either.
> Can you guess how many parachains does KodaDot supports? You can get exact answer on our Discord.
> Can you guess how much GitHub forks has KodaDot repository?
> Can you guess how much GitHub stars has KodaDot repository?
> Do you think KodaDot should have $koda token?
> Can you guess how much NFTs KodaDot is indexing from all parachains?
> Did you know that KodaDot has mobile app?
> Did you know KodaDot was among first NFT galleries supporting augmented reality for 3D objects, back in early January 2021, prior hacking in December 2019 😎?
> Did you know KodaDot had fuckups during early days? Yeah, all startups go through failures to learn from mistakes. Only in that way we knew, we are moving forward for better and quality product.
> Did you know, KodaDot has core values and ethos being distributed, decentralized and community owned? Have you tried participate in our ever growing community?
> Did you know KodaDot has replaced three logos in one month?
> Did you know that KodaDot's community translations was greatest growth hack for building community to participate? Then we've had in one week 16+ Telegram channel per language.
> Did you know KodaDot had 420 Discord members on their investors pitch deck? Such a coincidence.
> Did you know Kodadot founders encourage forking it's codebase and welcome more KodaDot instances in the wild? Come over to our Discord, we can help you with that and explain why it's important.
> Did you know that one of KodaDot founder was running infrastructure for eSports back in 2006?
> Did you know that these facts doesn't even matter? We just love to spend time with you till your action is finalized into Blockchain. Don't forget to drink enough water!
> Did you know we've crossed 420 opened issues in our backlog in GitHub?
> Did you know KodaDot promotes inclusive culture to be welcoming for widest spectrum of artist which actually participates on it's growth?
> We are in it for Building Better Worlds.
> Did you know if you make pull request to KodaDot repository, your fun fact can be shown here?
> Did you checked your JPEGs sales today?
> Have you minted something beautiful today?
> Did you know first version of KodaDot been looking worse than Craiglist in 1999?
> Did you know KodaDot founders won few Ethereum and Polkadot hackathons in past?
> Did you know KodaDot had to lay off few dozens investors interesting backing us?
> Did you know KodaDot cares about Climate change by planting trees in Portugal, called KodaForest?
> Did you know founders of KodaDot believe in simulated reality?
> Did you know you look beautiful today?

Add your best, **timeless** fun fact below.

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

loading-funfact-pr

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1436>
- [x] Implemented a textfield that shows on of 32 random fun facts while loading

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

File: utils/calculation.ts [hunk 1]
```diff
@@ -5,3 +5,8 @@ export function calculateUsdFromKsm(ksm: number, price: number): number {
 export function calculateKsmFromUsd(ksm: number, usd: number): number {
   return Math.ceil((usd / ksm) * 100) / 100;
 }
+
+// function includes the lower bound, but excludes the upper bound
+export function randomIntegerInRange(min: number, max: number): number {
```
Comment (vikiival) [inline utils/calculation.ts:10]:
```suggestion
export function randomIntegerBetween(min: number, max: number): number {
```

**Review by vikiival — COMMENTED** [review]
👁️

**Comment by vikiival** [comment]
Funky 🧙 

Screenshot 2021-12-08 at 15 39 42

File: utils/calculation.ts (hunk 1 above)
Comment (kkukelka) [reply utils/calculation.ts]: done 👍

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
@kkukelka payout sent https://kusama.subscan.io/extrinsic/0x7a63ea9060fb100f438e9ce9d8d6c38f355816e4a23002e5b59316400a19be7c

Let's make PR to `main` branch as well as I guess it will take some time to be on beta for while 😄

**Comment by yangwao** [comment]
#1460 seems I've found bug 🪲🐛

**Pull request merged**